README.rst

```diff
@@ -68,7 +74,7 @@ or as a directory — and you want to distribute it.
 Once your package is published, people can install it using *pip* just like
 any other package. In most cases, pip will download a 'wheel' package, a
 standard format it knows how to install. If you specifically ask pip to install
-an 'sdist' package, it will install and use Flit in a temporary environment.
+a 'sdist' package, it will install and use Flit in a temporary environment.
```

Comment:
You pronounce "sdist" starting with a consonant cluster? I would say S-dist, in which case "an" is the correct article. I don't know what the "official" pronunciation is.